pandas/tests/test_format.py

```diff
@@ -2952,6 +2952,16 @@ def test_to_csv_date_format(self):
         self.assertEqual(df_day.to_csv(), expected_default_day)
         self.assertEqual(df_day.to_csv(date_format='%Y-%m-%d'), expected_default_day)
 
+        # GH 7791
```

Comment:
``decimal`` is *only* for actual data blocks, not an index. I belive we only pass: ``date_format,quoting,float_format`` to the indexes.